scripts/zones/The_Garden_of_RuHmet/mobs/Kf_ghrah_blm.lua

```diff
@@ -0,0 +1,59 @@
+-----------------------------------
+-- Area: Grand Palace of Hu'Xzoi
+-- MOB:  Kf'ghrah BLM
+-----------------------------------
+require("scripts/globals/status");
+require("scripts/globals/magic"); -- no spells are currently set due to lack of info
+-----------------------------------
+-- OnMobSpawn Action
+-- Set core Skin and mob elemental bonus
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:AnimationSub(0);
+    mob:setLocalVar("roamTime", os.time());
+    mob:setModelId(1168); -- Dark
+    mob:addMod(MOD_DARKRES, 50);
+    mob:addMod(MOD_LIGHTRES, -50);
```

Comment:
https://www.youtube.com/watch?v=gvdf5n-zI14
never add or delete mods in onMobSpawn

add it in mob_spawn_mods instead